docs/integrations/gitlab/org.md

```diff
@@ -66,7 +168,13 @@ catalog:
         host: gitlab.com
         orgEnabled: true
         group: org/teams # Required for gitlab.com when `orgEnabled: true`. Optional for self managed. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
+        allowInherited: true # Allow groups to be ingested even if there are no direct members.
         groupPattern: '[\s\S]*' # Optional. Filters found groups based on provided pattern. Defaults to `[\s\S]*`, which means to not filter anything
+        schedule: # optional; same options as in TaskScheduleDefinition
```

Comment:
Is this true in all cases that it's optional? If I'm using the New Backend System would that not then be required?

Reply:
Hi @awanlin !

I changed this line. Could you please check? :)